lib/index.ts

```
@@ -304,7 +348,9 @@ async function getAllDeps(root: string, targetFile: string, options: Options):
     if (cleanupCallback) {
       cleanupCallback();
     }
-    return extractJsonFromScriptOutput(stdoutText);
+    const extractedJson = extractJsonFromScriptOutput(stdoutText);
+    extractedJson.versionBuildInfo = getVersionBuildInfo(gradleVersionOutput);
```

what is the parsing of build info fails? Will the whole test fail? Can we make it fail silently but log the failure?

---

yeah, I can make it fail silently.

---

for the `fail silently` bit, do I just use `debugLog()` or something else?

---

exactly add a `debugLog` and keep going so we can find this if it is run  with `-d`

---

wrapping `getVersionBuildInfo` in a try catch block and just doing a `debugLog` to avoid failing of `test`.